debian-mk-kernel.mk

```diff
@@ -4,6 +4,10 @@ ROOT := $(shell pwd)
 
 KERNEL_IMAGE=riscv-linux/vmlinux
 CROSS_COMPILE ?= /opt/riscv/bin/riscv64-unknown-linux-gnu-
+_CROSS_COMPILE=$(CROSS_COMPILE:-=)
+PREFIX=`basename ${_CROSS_COMPILE}`
+_CROSS_COMPILE_DIR=`dirname ${CROSS_COMPILE}`
+CROSS_COMPILE_DIR=$(shell echo ${_CROSS_COMPILE_DIR})
```

Comment:
I had hard time to understand why all that and how it works. Perhaps something like this would be better?
```
CROSS_COMPILE_DIR=$(abspath $(dir $(CROSS_COMPILE)))
``
(and get rid of `_CROSS_COMPILE`, `_CROSS_COMPILE_DIR` and `PREFIX` (see below)

Reply:
Hello @janvrany 

oh, it sounds good! I will try your suggestion!

Reply:
It works! I made commit!